scripts/unity/func_name_list.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+import sys
+import re
+import argparse
+
+def func_names_from_header(in_file, out_file):
+    with open(in_file) as f_in:
+        content = f_in.read()
+
+        with open(out_file,'w') as f_out:
+            x= re.findall(r"(struct\s|\s?)(\w+\s)(\*?)(\w+?)(\(.*?\))(;)", content, re.M | re.S)
```

Comment:
It is implied from the function name. But a comment like:

" Regex match all function names in the header file"

would be helpful